### DISCUSSION TITLE
Previewer for existing PDF

### DISCUSSION THREAD

**Discussion by DrWenz** [QuestPDF/QuestPDF#254] [Q&A] [answered]
Can we load a existing PDF to the previewer, so wen can use it as a PDF-Viewer?

**Comment by bennetbo** [comment, marked as answer]
No this is not possible, the previewer cannot render a PDF file. 
In short:
The previewer actually renders the drawing commands produced by QuestPDF with SkiaSharp (`SKCanvas`, `SKPicture`) and displays it as an 'image'. 
Unfortunately SkiaSharp only provides a way to render the content of an `SKCanvas` to a PDF file, not the other way around.

**Reply by DrWenz** [reply]
I checked the code and saw it, thanks for explanation.

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
